src/System/IO/BlockIO/URing.hs

```diff
@@ -71,7 +71,7 @@ setupURing URingParams { sizeSQRing, sizeCQRing } = do
           uringptr
           paramsptr
       params' <- peek paramsptr
-      when (fromIntegral sizeSQRing /= FFI.sq_entries params') $
+      when (fromIntegral sizeSQRing > FFI.sq_entries params') $
```

Comment:
Ok, so fail if the requested size is bigger than allocated, but allow the allocated to be bigger (since it rounds up to the next power of 2). Fine.

Reply:
Exactly